src/addons/attach/attach.js

```diff
@@ -11,12 +11,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = attach(require('../../dist/xterm'));
+    module.exports = attach(require('../../xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../dist/xterm'], attach);
+    define(['../../xterm'], attach);
```

Comment:
This ends up being places in out/addons/attach/attach.js for the tests and dist/addons/attach/attach.js for the actual build, I can do any verification you feel is necessary for this.